### DISCUSSION TITLE
layui table checkbox分页记忆功能

### DISCUSSION THREAD

**Discussion by liusongzhou** [layui/layui#1528] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.0

### 问题描述

当在第一页选中的纪录，切换到第二页就没有了，始终只能获取当前页选中的纪录。在保存或删除时，也就只能一页一页的去操作。有了记忆功能，就只需保存一次或删除一次。

### 业务代码

```auto
var checkStatus = table.checkStatus(id)
          var data = checkStatus.data; // 获取选中的数据
以上代码只能获取当前页的选中项记录，其他页已选中的记录获取不到。
```

### 浏览器

Chrome  版本 120.0.6099.130（正式版本） （64 位）

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
Gitee 搜索相似的问题：
https://gitee.com/layui/layui/issues/I3THO5
https://gitee.com/layui/layui/issues/I7DDCU
